backend/organization/utility/project.py

```diff
@@ -66,61 +67,44 @@ def create_new_project(data: Dict, source_language: Language) -> Project:
     return project
 
 
-def get_project_helpful_connections(project: Project, language_code: str) -> str:
-    if (
-        language_code != project.language.language_code
-        and project.translation_project.filter(
+def get_translation(
+    project: Project, language_code: str
+) -> Optional[ProjectTranslation]:
+    if project.language and language_code != project.language.language_code:
+        return project.translation_project.filter(
             language__language_code=language_code
-        ).exists()
-    ):
-        return project.translation_project.get(
-            language__language_code=language_code
-        ).helpful_connections_translation
+        ).first()
+    return None
+
+
+def get_project_helpful_connections(
+    project: Project, language_code: str
+) -> Optional[list[str]]:
```

Comment:
```suggestion
) -> Optional["list[str]"]:
```
To fix the error I mentioned above.